bp_common/src/include/bp_common_cache_service_if.vh

```diff
@@ -81,6 +81,8 @@ typedef enum logic [2:0] {
  e_cache_tag_mem_invalidate,
  // set tag and coherence state for given index and way_id
  e_cache_tag_mem_set_tag,
+ // set coherence state for given index and way_id
+ e_cache_tag_mem_set_state,
```

Comment:
Maybe we should get rid of invalidate, since it's strictly less useful than set_state?

Reply:
Makes sense. I think that should be a small change and I can roll into this.